lib/pattern-preset.js

```diff
@@ -77,6 +77,11 @@ export const presets = {
     ],
   },
 
+  RequirementsTxt: {
+    pathSubstrings: ['requirements.txt'],
+    githubClasses: [],
```

Comment:
Ah, that could allow it to work with alternative filenames like dev-requirements.txt (see  https://github.com/OctoLinker/browser-extension/pull/332#issuecomment-289582089), but I'm concerned that there might be false positives in other `type-text` blobs completely unrelated to python.